configuration.md

```diff
@@ -41,3 +43,14 @@ The result contains two tables:
   There are multiple rows per one document. All columns except `score` are part of the primary key.
 
   Note that the table also contains topic tags, marked as _tag_ in the `type` column.
+
+* `analysis-result-relations.csv` with relations-level results has the following columns:
+    * all `id` columns from the input table (used as primary keys)
+    * `type` type of the found relation, _VERB_ or _ATTR_, (primary key)
```

Comment:
I'm not sure whether keep the relation type as an internal thing or to publish it